source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -327,7 +334,8 @@ bool AggressiveDCEPass::AggressiveDCE(ir::Function* func) {
           // Mark stores as live if their variable is not function scope
           // and is not private scope. Remember private stores for possible
           // later inclusion
-          if (IsVarOfStorage(varId, SpvStorageClassPrivate))
+          if (IsVarOfStorage(varId, SpvStorageClassPrivate) ||
+              IsVarOfStorage(varId, SpvStorageClassWorkgroup))
```

Comment:
Why not call IsLocalVar() here?

Reply:
Ah, ok.  Could you add a small comment here?

Reply:
Done.